app/(main)/settings/team/team-members-list.tsx

```diff
@@ -30,12 +27,12 @@ export function TeamMembersList({
 			<div className="divide-y divide-zinc-800">
 				{members.map((member) => (
 					<TeamMemberListItem
-						key={member.userId}
+						key={`${teamDbId}-${member.userId}`}
```

Comment:
IMO:

Using `teamDbId` in client-side code is not ideal, but since it's only being used as a key for the DOM, I think it's fine to leave it as is.

Reply:
This is about using public id and db id, right? I too thought that the public id was originally needed, but since it is removed by key, I figured it was no problem 👌